app/frontend/src/assets/scss/style.scss

```diff
@@ -10,14 +24,17 @@ h3 { font-size: 1.444em; /* (25.92px) */ }
 h4 { font-size: 1.2em; /* (21.60px) */ }
 
 h1, h2, h3, h4, h5 {
+  color: #494949;
```

Comment:
Once we expand on our theming knowledge we can make stuff like #494949 SASS variables (could do in the existing file anyways, but would leave it till we determine how we want to set Vuetify SASS detaults in a theme)